packages/devtools_app/devtools_app.iml

```diff
@@ -9,6 +9,27 @@
       <excludeFolder url="file://$MODULE_DIR$/.idea" />
       <excludeFolder url="file://$MODULE_DIR$/.pub" />
       <excludeFolder url="file://$MODULE_DIR$/build" />
+      <excludeFolder url="file://$MODULE_DIR$/linux/flutter/ephemeral/.plugin_symlinks/file_selector_linux/.dart_tool" />
```

Comment:
why are these being added? Since all of these plugins were added a long time ago, I would think these entries would already be here if they were needed

Reply:
Accidental commit on my end. Will revert (not sure why these files get updated or aren't part of .gitignore).